bin/rails

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../../config/application', __FILE__)
-require_relative '../config/boot'
-require 'rails/commands'
+# frozen_string_literal: true
```

Comment:
Good catch! I'm surprised I let this through. I've since reconfigured bundler to stop doing that, but let me get rid of these ;)